battenberg/core.py

```diff
@@ -33,6 +35,16 @@ def __init__(self, repo: Repository):
     def is_installed(self) -> bool:
         return TEMPLATE_BRANCH in self.repo.listall_branches()
 
+    def fetch_remote_template(self):
```

Comment:
Should public methods like this have docstrings?

Reply:
Good point. Explicitly updated the methods to prefix private methods with `_` and added docstrings to all public methods using Google Style Guide format :)